fixtures/keys/root

```diff
@@ -0,0 +1 @@
+2a56071cdc567ebf897df1bf21f00d2a@@@@100000@@@@6dbc945dd60f2a924116e48887c457b17da412f32d6529b770a3f95daaeedd83@@@@bf607649a08aaabe0764b5e53b7211e1@@@@f2f0717d46b2a8b49411553df1d4aae8e2e6ffaca6bd5090f2d3fe5bfa6447267eb13c1a76d55a1001d30efba9db1f7c51b854362618d14a5dd7a55f4faa4f1f8244d1094211106fa9da53daf6146d82cf0b6e42c357c01cbe0fddb6fa4d8a93134f5231a1204ae4da8a623018867a250ca4c03a619971d92dce28c5d276b292209bdb990c1f30684053da70460b898386afcf0b34dd9ae810e111ec4761306a0fb520c4d28099c8806330051584e7fa8ad1c9901c99c41ce4b63f1e4cb57b1d2e9810d80e523a7912291624bb2c95d2d49601dd3e79b1612c924755ff3a362d35b10a03c489a6ce47045200cb1ea9d9a75141f13a8bf7be05453ddb13eb64bc0bfa5c6873d2b410e2d90d60e6bce6b403e4cbcb7bc874dfde114dba8376b1a4d3c1d788952300a29c6e6c9d70faadb2a88467c47e0abbab6d073b7e42b9389162c2526b3964e55e40c07be160c0ce4a
```

Comment:
Do we need any of the files in this directory to actually be added?

Reply:
I think we do, because the `generate.py`  uses them as the signing keys for the four main roles. I don't know how to generate those out of thin air (or at least, `generate_fixtures.py` didn't obviously contain a way to do it). I'd be open to doing that in a follow-up though.

Reply:
I raised this question with the TUF upstream, and we concluded that a fixed set of keypairs (which any fixture generator can use for any purpose) was the best way to achieve determinism here (as compared to a shared seed, which would have required invasive changes or patching of the TUF library).